qutip/tests/core/test_numpy_backend.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import numpy as np
+from unittest.mock import patch, MagicMock
+
+from ...settings import settings
+from ...core.numpy_backend import NumpyBackend
+
+# Mocking JAX and NumPy to demonstrate backend switching
+mock_jax = MagicMock()
+mock_np = MagicMock()
+
+
+class TestNumpyBackend:
+    def test_backend_property(self):
+        settings.core["numpy_backend"] = mock_np
+        np_backend = NumpyBackend()
```

Comment:
We use it as `from .core.numpy_backend import np`, so it should be tested that way.